caravel/assets/javascripts/explorev2/components/ChartContainer.jsx

```diff
@@ -14,6 +15,13 @@ const propTypes = {
 };
 
 class ChartContainer extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      selector: `#${this.props.sliceContainerId}`,
```

Comment:
sorry I created the world_map branch on top of this branch since there are some functions needed from here. I'll merge this first then rebase on the world_map one